### ISSUE TITLE
Some problems about the results

### ISSUE THREAD

**Issue by Shuweis** [zhuhancheng/MetaIQA#4]
Hi!Congratulations to your work!
In the process of reproducing the code, I trained the prior knowledge model on TID2013 and Kadid10k, and fine-tuned on the CLIVE data set, but the SROCC was about 0.48, which did not achieve the results in the paper. Do you get similar results? I didn't make any change but the results were not ideal.

**Comment by zhuhancheng** [comment]
This issue has been fixed by commenting out 'labels= labels / 100.0'.

**Comment by seyeeet** [comment]
can you share what did you use for 'test_image.csv and 'train_image.csv'
